sources/packages/web/src/components/common/AnnouncementBanner.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-alert
```

Comment:
We already have a banner component, `Banner.vue`. 

Is there a specific reason that we are not re-using it?

Reply:
Yes, the generic banner component did not follow the requested display.

Reply:
The component in place (`Banner.vue`) should be used.

Reply:
reworked